<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientsDialog</class>
 <widget class="QDialog" name="ClientsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Klijenti</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QTableWidget" name="clientsTableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <column>
      <property name="text">
       <string>Naziv</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>OIB</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Adresa</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Telefon</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Email</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <widget class="QPushButton" name="addClientButton">
       <property name="text">
        <string>Dodaj</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editClientButton">
       <property name="text">
        <string>Uredi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteClientButton">
       <property name="text">
        <string>Obri≈°i</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Zatvori</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
